tests/impact/client/operations/test_external_result.py

```diff
@@ -1,70 +0,0 @@
-import pytest
```

Comment:
Was this file meant to be removed without adding any corresponding tests for the operation?

Reply:
Added the tests back :)